[2024-07-18 06:34:27,796] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: xcom_example_dag.pull_task 2023-12-23T00:00:00+00:00 [queued]>
[2024-07-18 06:34:27,825] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: xcom_example_dag.pull_task 2023-12-23T00:00:00+00:00 [queued]>
[2024-07-18 06:34:27,831] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-18 06:34:27,833] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2024-07-18 06:34:27,833] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-18 06:34:27,879] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): pull_task> on 2023-12-23T00:00:00+00:00
[2024-07-18 06:34:27,890] {standard_task_runner.py:52} INFO - Started process 2613 to run task
[2024-07-18 06:34:27,913] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_example_dag', 'pull_task', '2023-12-23T00:00:00+00:00', '--job-id', '344', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom.py', '--cfg-path', '/tmp/tmp3mo9si9n', '--error-file', '/tmp/tmpxctixp57']
[2024-07-18 06:34:27,919] {standard_task_runner.py:77} INFO - Job 344: Subtask pull_task
[2024-07-18 06:34:28,087] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_example_dag.pull_task 2023-12-23T00:00:00+00:00 [running]> on host a868c7bacc12
[2024-07-18 06:34:28,224] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_example_dag
AIRFLOW_CTX_TASK_ID=pull_task
AIRFLOW_CTX_EXECUTION_DATE=2023-12-23T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-23T00:00:00+00:00
[2024-07-18 06:34:28,244] {logging_mixin.py:104} INFO - Pulled message: Hello from push_function!
[2024-07-18 06:34:28,245] {python.py:118} INFO - Done. Returned value was: None
[2024-07-18 06:34:28,275] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=xcom_example_dag, task_id=pull_task, execution_date=20231223T000000, start_date=20240718T063427, end_date=20240718T063428
[2024-07-18 06:34:28,330] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-18 06:34:28,361] {local_task_job.py:146} INFO - Task exited with return code 0
